vendor/assets/javascripts/util.js

```diff
@@ -0,0 +1,7 @@
+Cmsify.util = {
+  filterAuthenticityToken: function(arr) {
+    return arr.filter(function(element) {
+      return element.name !== "authenticity_token";
+    });
+  }
+}
```

Comment:
I didn't want to break nested_resource, which I imagine requires the `authenticity_token`